streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -1354,7 +1354,7 @@ class ZStream[-R, +E, +A](val process: ZManaged[R, E, Pull[R, E, A]]) extends Se
         add <- self
                 .mapM(f)
                 .distributedDynamicWith(
-                  buffer, { kv: (K, V) =>
+                  buffer, { (kv: (K, V)) =>
```

Comment:
This is highlighted as redundant by Intellij, someone might undo it by mistake later - a way around it would be to use parens instead of curlies to enclose this lambda. (actually, I think it's legal in this case to avoid even parens, since the argument is enclosed)

Reply:
Thanks! I took out the brackets and parentheses entirely.